### PR TITLE
Updated csr behavior and store fault behavior

### PR DESCRIPTION
**Pull request by ved-rivos**

### CODE REVIEW

File: cfi_backward.adoc [hunk 1]
```diff
@@ -44,8 +45,42 @@ using the `mop.r` and `mop.rr` instructions defined by the Zimops extension.
...
+The term `xBCFIACT` is used to determine if backward-edge CFI is active at a
+privilege level `x` and is defined as follows:
+
+.`xFCFIACT` determination
+[source, text]
+----
+if ( privilege == M-mode ) 
+    xBCFIACT = 1
+else if ( menvcfg.CFI == 1 && V == 0 && privilege == S-mode )
+    xFCFIACT = mstatus.SBCFIEN
```
Comment (deepak0414) [inline cfi_backward.adoc:69]: xBCFIACT

File: cfi_backward.adoc [hunk 2]
```diff
@@ -44,8 +45,42 @@ using the `mop.r` and `mop.rr` instructions defined by the Zimops extension.
...
+else if ( menvcfg.CFI == 1 && V == 0 && privilege == U-mode )
+    xFCFIACT = mstatus.UBCFIEN
```
Comment (deepak0414) [inline cfi_backward.adoc:71]: xBCFIACT

File: cfi_backward.adoc [hunk 3]
```diff
@@ -44,8 +45,42 @@ using the `mop.r` and `mop.rr` instructions defined by the Zimops extension.
...
+else if ( menvcfg.CFI == 1 && henvcfg.CFI == 1 & V == 1 && privilege == S-mode )
+    xFCFIACT = vsstatus.SBCFIEN
```
Comment (deepak0414) [inline cfi_backward.adoc:73]: xBCFIACT

File: cfi_backward.adoc [hunk 4]
```diff
@@ -44,8 +45,42 @@ using the `mop.r` and `mop.rr` instructions defined by the Zimops extension.
...
+else if ( menvcfg.CFI == 1 && henvcfg.CFI == 1 & V == 1 && privilege == U-mode )
+    xFCFIACT = vsstatus.UBCFIEN
```
Comment (deepak0414) [inline cfi_backward.adoc:75]: xBCFIACT

**Review by deepak0414 — APPROVED** [review]

File: cfi_backward.adoc [hunk 5]
```diff
@@ -44,8 +45,42 @@ using the `mop.r` and `mop.rr` instructions defined by the Zimops extension.
...
+When `henvcfg.CFI` is 0, then Zicfi is not enabled for use when `V=1`.
+
+When `V=1` and `menvcfg.CFI` and `henvcfg.CFI` are both 1, then backward-edge CFI
+is active at S-mode if `vsstatus.SBCFIEN` is 1 and is active at U-mode if
+`vsstatus.UBCFIEN` is 1.
+
+The term `xBCFIACT` is used to determine if backward-edge CFI is active at a
+privilege level `x` and is defined as follows:
+
+.`xFCFIACT` determination
```
[the remark this answers is not in the document]
Comment (ved-rivos) [reply cfi_backward.adoc]: yes. fixed

**Pull request merged**